pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -572,6 +572,17 @@ func waitForQuorumWithMons(context *clusterd.Context, clusterName string, mons [
 			<-time.After(time.Duration(sleepTime) * time.Second)
 		}
 
+		// wait for the mon pods to be running
+		running, err := k8sutil.PodsRunningWithLabel(context.Clientset, clusterName, "app="+appName)
```

Comment:
Did this change slip in?
It does not seem directly related to "making the integration tests more resilient" as it is changing "core" mon logic, which I don't think was a problem in the integration tests or not?